Allow to trigger a remote query of history events for exchanges (#5924)
rotki/rotki: docs/websockets.rst

- ``service``: Service for which an API key is needed.
- ``location``: For services like etherscan that require different API keys for different locations this is the location for which the API key is needed. If a service does not require a location then this key will be missing.

Query new history events
=========================

> In addition to history from evm transactions we need to query events from exchanges that support this kind of events. The backend will emit a ws message as the following when a query start for such exchange.

Reviewer: ```suggestion
In addition to history from evm transactions we need to query events from exchanges, eth staking events etc. The backend will emit a ws message as the following when such a query starts.
```

::

    {
        "type": "querying_events_status",
        "data": {
            "status": "querying_events_started",

>             "location": "kraken",

Reviewer: Should it be location or also even type? Since location alone won't tell us much for eth withdrawals or eth block production

            "name": "My kraken exchange"
        }
    }

> - ``status``: Can be either `querying_events_started`, `querying_events_finished`, `querying_events_status_update`. Each pair of events is triggered per exchange instance.

Reviewer: Here in all of these the description seems to be specific to exchanges. I think we need to change this to make it generic enough for when adding WS progress for other event type queries.
